nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/ElasticsearchRestProcessor.java

```diff
@@ -25,12 +25,15 @@
 import org.apache.nifi.processor.ProcessContext;
 import org.apache.nifi.processor.ProcessSession;
 import org.apache.nifi.processor.util.JsonValidator;
+import org.apache.nifi.processor.Relationship;
 import org.apache.nifi.processor.util.StandardValidators;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 
-public interface ElasticSearchRestProcessor {
+public interface ElasticsearchRestProcessor {
```

Comment:
That'll wreck havoc on case-insensitive file systems. On this MBP I'm using, I've already had to clone the repo into a case-sensitive disk image just to complete the PR because HPFS+ was screwing with Git after this change.